pkg/executables/kubectl.go

```diff
@@ -825,3 +825,116 @@ func (k *Kubectl) GetBundles(ctx context.Context, kubeconfigFile, name, namespac
 
 	return response, nil
 }
+
+func (k *Kubectl) PatchTolerations(ctx context.Context, taints []corev1.Taint, resource string, name string, kubeconfigFile string, namespace string, path string) error {
```

Comment:
This method signature is a bit weird, patching tolerations but passing taints as an argument